examples/tutorials/introduction/0_introduction.py

```diff
@@ -0,0 +1,113 @@
+"""
+Introduction to Traits
+======================
+
+To make the tutorial more practical, let's imagine that you are
+writing some tools to organize and process image data, perhaps
+to help you develop a machine learning algorithm, or to document
+your research.  This image data might come from a camera, a scanning
+electron microscope, or from some more exotic source.  No matter the
+source, the image will have data associated with it, such as who
+produced the image, the equipment that was used, the date and time
+that the image was taken, and so forth.
+
+In this tutorial, we'll consider an example where we have a collection
+of greyscale SEM images, each of them stored in a file, along with other
+information, such as the sample that was imaged, the size of the scanned
+region, and the operator who took the image.
+
+We would like to read in the image and provide a number of standard
+analysis steps to for the user, as well as making the raw image data
+available as a NumPy array.
+
+The sample code here shows how you might create a class in standard
+object-oriented Python with NumPy for analysis and Pillow for image
+loading.  Modify the code in the sample and see how you can use it.
+
+In the tutorial, we'll look at how we can use Traits to simplify and
+improve the code here.
```

Comment:
It might be helpful to mention upfront that one should install the following in order to run the entire tutorial:
```
pip install traits pillow numpy traitsui pyface pyqt5
```
Suppose these tutorials are run from the demo app (which requires GUI dependencies), pillow and numpy are needed in addition.

Reply:
There's a bigger question around deployment of the demo apps.  At a minimum I was thinking that we should add something to etstool which creates an environment with the right dependencies and invokes the tutor.py script.